product_assortment/security/product_assortment_security.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="group_product_assortment_manager" model="res.groups">
```

Comment:
If the intention is to restrict the `Product Assortment` menu to managers, I think it’s necessary to add the new group to the button in the Partner form view as well. Or did I misunderstand the PR's purpose? What do you think?
![image](https://github.com/user-attachments/assets/8ed78313-7e3a-4028-a72d-a9e7216091a8)